src/main/java/seedu/address/model/AddressBook.java

```diff
@@ -127,4 +128,8 @@ public boolean equals(Object other) {
     public int hashCode() {
         return persons.hashCode();
     }
+
+    public void addTeacher(Teacher teacher) {
+        addPerson(teacher);
```

Comment:
Missing JavaDoc for this public method?

Reply:
noted, will add  javadoc for this in my next commit